### PR TITLE
FEAT: adds profile, improve layout

### PR DESCRIPTION
**Pull request by usrrname**
# Changes

- adds a myspace type profile
image

- adds status components
- returns landing page at `/`

### CODE REVIEW

File: src/layout/main.tsx [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
+          <pre>
+            {`
+          +----------+
+          |  ___   o |
+          | /_|_\\    |
+          |~~~|~~~~~~|
+          |~~~|~~~~~~|
+          |   | \\__  |
+          +----------+
+          `}
```
Comment (evert) [inline src/layout/main.tsx:29]: This is now broken probably =( space is significant in `<pre>` This is why i put it in a variable first.

File: src/templates/member-profile.tsx [hunk 2]
```diff
@@ -1,14 +1,111 @@
...
 }
 
-export function MemberProfile(props: Props) {
+export function MemberProfile(props: MemberProps) {
+  const { name, pronouns, yoe, avatar, bio, website, githubHandle, status, interests, music, heroes, books, friends } = props;
+  const githubUrl = `https://github.com/${githubHandle}`;
+
+  return (
+    <>
+      <html>
```
Comment (evert) [inline src/templates/member-profile.tsx:34]: why did you delete the layout?

**Comment by evert** [comment]
But good to merge anyway =)

File: src/templates/member-profile.tsx (hunk 2 above)
Comment (usrrname) [reply src/templates/member-profile.tsx:34]: Extreme separation of concerns initially 😆

File: src/layout/main.tsx (hunk 1 above)
Comment (usrrname) [reply src/layout/main.tsx:29]: it should look like 
Screenshot 2024-05-18 at 4 47 00 PM

Comment (evert) [reply src/layout/main.tsx:29]: So my original idea was to build an ascii-only profile so we could also rehost this on the Gopher protocol, so the avatar should be part of the user type probably!

Comment (evert) [reply src/layout/main.tsx:29]: But anyway this is yours now so go right ahead!

Comment (usrrname) [reply src/layout/main.tsx:29]: Was it supposed to animate or...?

**Comment by usrrname** [comment]
...wait!!! _screams into the merge tunnel_

**Comment by evert** [comment]
Sorry @usrrname didn't want to block

**Comment by usrrname** [comment]
No worries @evert I think I was just being a perfectionist
